.changelog/3745.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:note
+resource/cloudflare_logpush_job: Deprecate Remove frequency in cloudflare_logpush_job
```

Comment:
```suggestion
resource/cloudflare_logpush_job: Deprecate `frequency` in favour of `max_upload_interval_seconds`
```

Reply:
Oh, sorry. Copy-paste error. Thanks for fixing!